src/__tests__/screens/movieScreen.test.tsx

```diff
@@ -56,15 +56,16 @@ describe('Movie Screen Test Suite', async () => {
             initialEntries: ['/search?q=iron+man']
         });
         // render screens
-        render(<RouterProvider router={router}/>);
+        render(<RouterProvider router={router} />);
 
         await waitFor(() => screen.getByTestId('search-results-heading'));
         expect(screen.getAllByRole('heading')[0]).toHaveTextContent('Search Results Page');
         // click on show card to change screen
         await user.click(screen.getAllByTestId('show-details-link')[0]);
         await waitFor(() => screen.getByTestId('show-details-heading'));
-        expect(screen.getByTestId('details-provider')).toBeInTheDocument();
-        expect(screen.getByTestId('details-provider')).toHaveTextContent('Disney Plus');
+        const image: HTMLImageElement = screen.getByAltText('Disney Plus logo');
+        expect(image).toBeInTheDocument();
+        expect(image.src).toContain('https://image.tmdb.org/t/p/w500/7rwgEs15tFwyR9NPQ5vpzxTj19Q.jpg');
```

Comment:
I imagine this is not a big deal, but does `toContain` match a pattern? For example, would:
```js
expect('abcdef').toContain('abc');
```
be true? If so, `toBe` probably makes more sense.